<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="Tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Page1">
       <attribute name="title">
        <string>Aus Tabelle erstellen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="30" column="1" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="getList">
            <property name="text">
             <string>Aktuelle Liste laden</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readList">
            <property name="text">
             <string>Account einlesen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="35" column="2">
         <widget class="QCheckBox" name="automatic_continue">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>weiter</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="34" column="1" colspan="3">
         <widget class="QLabel" name="error_field">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="29" column="1" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Zeile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backward">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>⬅</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="row_field">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forward">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>➡</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="list_field">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>∞</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="35" column="3">
         <widget class="QPushButton" name="buttonNew">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Account erstellen</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Stammdaten</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lname">
             <property name="placeholderText">
              <string>Nachname</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="telephone">
             <property name="placeholderText">
              <string>Telefon</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="uid">
             <property name="placeholderText">
              <string>ID (Standard: it, - für leer)</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="fname">
             <property name="placeholderText">
              <string>Vorname</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="mail">
             <property name="placeholderText">
              <string>E-Mail</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="role">
             <property name="placeholderText">
              <string>Rolle</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="facility">
             <property name="placeholderText">
              <string>Einrichtung</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="16" column="1" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Gruppen</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="0">
              <widget class="QCheckBox" name="it_support">
               <property name="text">
                <string>IT-Support</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="diversity">
               <property name="text">
                <string>Diversity</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="dig_marketing">
               <property name="text">
                <string>Digitales Marketing</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="vertrieb_verkauf">
               <property name="text">
                <string>Vertrieb und Verkauf</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="grafikdesign">
               <property name="text">
                <string>Grafikdesign</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="kundenservice">
               <property name="text">
                <string>Kundenservice</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="finanzanalyse">
               <property name="text">
                <string>Finanzanalyse</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QCheckBox" name="dynamics">
               <property name="text">
                <string>Dynamics 365</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="soft_skills">
               <property name="text">
                <string>Soft Skills</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="dig_arbeitsplatz">
               <property name="text">
                <string>Digitaler Arbeitsplatz</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QCheckBox" name="it_administration">
               <property name="text">
                <string>IT-Administration</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QCheckBox" name="projektmanagement">
               <property name="text">
                <string>Projektmanagement</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="empfang">
               <property name="text">
                <string>Empfang</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="jobcoaching">
               <property name="text">
                <string>Jobcoaching</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QCheckBox" name="power_platform">
               <property name="text">
                <string>Power Platform</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="mc_af">
               <property name="text">
                <string>Azure</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="mc_adf">
               <property name="text">
                <string>Azure Data</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QCheckBox" name="mc_aaf">
               <property name="text">
                <string>Azure AI</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QCheckBox" name="softwareentwicklung">
               <property name="text">
                <string>Softwareentwicklung</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QCheckBox" name="datenanalyse">
               <property name="text">
                <string>Datenanalyse</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QCheckBox" name="microsoft">
               <property name="text">
                <string>Microsoft 365</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QCheckBox" name="teams">
               <property name="text">
                <string>Microsoft Teams</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="37" column="1" colspan="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="33" column="1" colspan="3">
         <widget class="QProgressBar" name="progress">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Page2">
       <attribute name="title">
        <string>Account bearbeiten</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>401</width>
          <height>641</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="8" column="0">
          <widget class="QTextBrowser" name="status_field"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="tempPasswd">
           <property name="text">
            <string>temporäres Passwort</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="changePasswd">
           <property name="text">
            <string>Passwort ändern</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="selfChosenPasswd">
           <property name="placeholderText">
            <string>Passwort eingeben,  leer für automatisch erstellen</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="textfield">
           <property name="text">
            <string>Ausgabe:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="deleteAccount">
           <property name="text">
            <string>Account löschen</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="search">
           <property name="text">
            <string>Account suchen</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="searchField">
           <property name="placeholderText">
            <string>Name des Accounts</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Einstellungen">
    <property name="title">
     <string>&amp;Einstellungen</string>
    </property>
    <addaction name="action_Zugangsdaten"/>
   </widget>
   <addaction name="menu_Einstellungen"/>
  </widget>
  <action name="action_Zugangsdaten">
   <property name="text">
    <string>&amp;Zugangsdaten</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
